styles.css

```diff
@@ -0,0 +1,95 @@
+body {
+  width: 310px;
+}
+
+.nav-bars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 48px;
+  width: 340px;
+}
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  width: 350px;
+  height: 470px;
+  padding-bottom: 50px;
+  padding-top: 50px;
+}
+
+.myself {
+  display: flex;
+  flex-direction: column;
+  background-image: url(./photo/bg/header-shapes-mobile.png);
+  padding-top: 20px;
+  padding-right: 24px;
+  width: 400px;
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: top;
+}
+
+.myself > .greeting {
+  align-items: center;
+  width: 200px;
+  height: 112.75px;
+  color: #172b4d;
+  padding-left: 23px;
+  padding-top: 30px;
+  padding-bottom: 20px;
+}
```

Comment:
[ OPTIONAL ] I noticed here that you can fix the spaces the padding and the height so match the Figma template 🔥🔥